Update to 6.3.0

conda-forge-admin opened the pull request:
Hi! This is the friendly automated conda-forge-webservice.

I've started rerendering the recipe as instructed in #29.

If I find any needed changes to the recipe, I'll push them to this PR shortly. Thank you for waiting!



Here's a checklist to do before merging.
- [ ] Bump the build number if needed.

conda-forge-webservices[bot] committed:
dummy commit for rerendering

Commit:
MNT: Re-rendered with conda-build 3.27.0, conda-smithy 3.28.0, and conda-forge-pinning 2023.11.10.07.51.14

hmaarrfk committed:
Update meta.yaml

hmaarrfk committed:
Update meta.yaml

hmaarrfk committed:
Delete recipe/6.2.0-arm.patch

hmaarrfk committed:
try the url again????

hmaarrfk commented:
i feel like they blocked azure's cloud since I can download it and build fine locally...

isuruf committed:
Use GNU FTP mirror

GMP maintainers are blocking Github servers because
of a DOS from a heavy Github action.

hmaarrfk commented:
Thanks isuru
